public/src/components/channelManagement/bannerTests/bannerVariantPreview.tsx

```diff
@@ -120,17 +119,16 @@ const BannerVariantPreview: React.FC<BannerVariantPreviewProps> = ({
     window.guardian.automat = {
       react: React,
       preact: React,
-      emotionCore,
-      emotionTheming,
-      emotion,
+      emotionReact,
+      emotionReactJsxRuntime,
     };
 
     const stage = getStage();
 
     const baseUrl =
       stage === 'PROD'
-        ? 'https://contributions.guardianapis.com/modules/v1/banners'
-        : 'https://contributions.code.dev-guardianapis.com/modules/v1/banners';
+        ? 'https://contributions.guardianapis.com/modules/v2/banners'
```

Comment:
doesn't have to be now, but it would be nice to move the version into a shared const somewhere